repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -13,7 +13,8 @@
 REPOSITORIES_MAPPING = {
     'rhel8-appstream': 'rhel-8-for-x86_64-appstream-rpms',
     'rhel8-baseos': 'rhel-8-for-x86_64-baseos-rpms',
-    'rhel8-crb': 'codeready-builder-for-rhel-8-x86_64-rpms'}
+    'rhel8-crb': 'codeready-builder-for-rhel-8-x86_64-rpms',
+    'rhel8-supplementary': 'rhel-8-for-x86_64-supplementary-rpms'}
```

Comment:
This will become obsoleted soon as this is just architecture specific name.

Reply:
Yeah, it will be taken care of in another PR.